### ISSUE TITLE
pip3 install fbchat fails

### ISSUE THREAD

**Issue by stuartlmorris** [carpedm20/fbchat#219]
`pip3 install fbchat`

results in the following error:

> Collecting fbchat
>   Using cached fbchat-1.0.25.tar.gz
>     Complete output from command python setup.py egg_info:
>     Traceback (most recent call last):
>       File "<string>", line 1, in <module>
>       File "/tmp/pip-build-us9itz1q/fbchat/setup.py", line 14, in <module>
>         readme_content = f.read().strip()
>       File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
>         return codecs.ascii_decode(input, self.errors)[0]
>     UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 1009: ordinal not in range(128)
>     
>     ----------------------------------------
> Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-us9itz1q/fbchat/

when installed on Python 3.6 (Ubuntu 17.10). It appears that it is the Copyright symbol on the last line of the README.rst file that is causing the problem. It appears to happen in all 1.0.2x releases.

**Comment by madsmtm** [comment]
Could you perhaps try adding:
```
# -*- coding: UTF-8 -*-
from __future__ import unicode_literals
```
at the top of your `setup.py`? If that helps, I'll add it to the main branch.

**Comment by madsmtm** [comment]
Okay, I had the same issue on my machine just now, fixed in `1.1.1`
